docker/ci/Dockerfile

```diff
@@ -19,8 +19,9 @@ ENV CHAIN="/go/src/chain" \
 ADD install /usr/bin
 
 RUN yum -y update \
-    && yum install -y autoconf automake bash curl gcc gcc-c++ git libtool make openssl-devel ruby wget unzip \
+    && yum install -y autoconf automake bash bzip2 bzip2-devel curl gcc gcc-c++ git libtool make openssl-devel snappy snappy-devel ruby tar wget unzip \
```

Comment:
Ugh, I'm sorry I didn't catch this earlier, but I bet we don't need bzip or snappy here, either.